src/splitUrl.js

```diff
@@ -0,0 +1,13 @@
+function toSplitUrl(query) {
+  const text = query[0].slice(1);
```

Comment:
Instead of using array indexing to access query parameters, use destructuring assignment. You can change it to: `const [text, targetCase] = query;` and then remove lines 2 and 3.